extension/src/plots/model/index.ts

```diff
@@ -237,7 +242,14 @@ export class PlotsModel extends ModelWithPersistence {
       : this.getSelectedTemplatePlotData(plotId)
 
     try {
-      writeJson(filePath, rawData as unknown as Record<string, unknown>, true)
+      if (type === PlotExportType.JSON) {
+        writeJson(filePath, rawData as unknown as Record<string, unknown>, true)
+      } else {
+        await writeCsv(
+          filePath,
+          rawData as unknown as Array<{ [key: string]: unknown }>
```

Comment:
[Q] How can the data types for the raw data be different depending on the type of export?

Reply:
These types were actually just a bandaid so I could test exports. Will adjust the types!